packages/element-plus/make-installer.ts

```diff
@@ -5,17 +5,16 @@ import type { ConfigProviderContext } from '@element-plus/tokens'
 
 const INSTALLED_KEY = Symbol('INSTALLED_KEY')
 
-declare module '@vue/runtime-core' {
-  interface App {
-    [INSTALLED_KEY]?: boolean
-  }
+interface ElementPlusApp extends App {
+  [INSTALLED_KEY]: boolean
 }
 
 export const makeInstaller = (components: Plugin[] = []) => {
   const install = (app: App, options?: ConfigProviderContext) => {
```

Comment:
Forgot about it.
Just change it to `any`. I think the type of `app` is not important.
```suggestion
  const install = (app: any, options?: ConfigProviderContext) => {
```